apps/web/src/components/ui/password-form.tsx

```
@@ -0,0 +1,111 @@
+    return (
+      <div
+        className={cn(
+          "flex w-[400px] flex-col items-start justify-center gap-1 rounded-[10px]",
+          className,
+        )}
+      >
+        {label && (
+          <label
+            className={cn(
+              "font-medium text-ink-black",
+              isDesktop ? "text-base leading-[26px]" : "text-sm leading-[23px]",
```

ESLint

[error] Delete `,`

(prettier/prettier)